ui.R

```diff
@@ -172,6 +180,86 @@ shinyUI(fluidPage(
             )
           )
         ),
+        tabPanel(
+          "Description",
+          mainPanel(
+            h3("Measures of work incentives:"),
+            tags$ul(
+              tags$li(strong("Net Income"), " – The Net Income tab shows how an individual’s income in the hand, that is income after taxes and government welfare transfers, changes as their earnings from work change."),
+              br(),
+              tags$li(withMathJax(strong("Effective Marginal Tax Rate (EMTR)"), " – The EMTR tab shows what percentage of a one dollar increase in gross earnings is not received in the hand due to taxation and the abatement of government welfare transfers.
+                                  $$EMTR=1-\\frac{\\text{change in income in hand}}{\\text{change in gross earnings}}$$")),
+              tags$li(strong("Replacement Rate (RR)"), " – The RR tab shows what percentage of an individual’s income in the hand would be replaced by government welfare transfers if they chose to stop working.",
+                      span(title = "If someone earned $1,000 per week at their job and receives $600 in unemployment benefits when they stop working, their replacement rate would be 60%. A higher replacement rate means people can receive benefits that are closer to their work income, which may reduce their incentives to remain in employment. Replacement rates are also used to evaluate the adequacy of social safety nets and to compare welfare systems across different countries or regions.",
+                           class = "tooltip-span",
+                           "Example"),
+                      "$$RR=\\frac{\\text{income in the hand when not working}}{\\text{income in the hand when working}}$$"),
+              tags$li(strong("Participation Tax Rate (PTR)"), " – The PTR tab shows the total amount of additional tax paid and welfare transfers forgone by an individual as they move from unemployment into work, as a percentage of gross income.",

```

Comment:
Suggest this sentence, for more consistency with the EMTR & RR sentences.
```suggestion
              tags$li(strong("Participation Tax Rate (PTR)"), " – The PTR tab shows what percentage of gross earnings is not received in the hand when an individual moves from unemployment into work, due to additional tax paid and reduced government welfare transfers.",
```